stacks/root.yml

```diff
@@ -948,7 +945,7 @@ Resources:
         KeyPairName: !Ref AsgKeyPairName
         RootStackName: !GetAtt Constants.RootStackName
         RootStackId: !GetAtt Constants.RootStackId
-        CastlePostgresClientSecurityGroupId: !GetAtt Apps200AStack.Outputs.CastlePostgresClientSecurityGroupId
+        CastlePostgresClientSecurityGroupId: !If [CreateApplicationStacks, !GetAtt Apps200AStack.Outputs.CastlePostgresClientSecurityGroupId, ""]
```

Comment:
Will this work, or will CFN be angry that a `AWS::EC2::SecurityGroup::Id` parameter gets set to a blank string?
```

Reply:
Can't say for sure